_posts/2016-01-06-format-support.md

```diff
@@ -0,0 +1,115 @@
+---
+layout: post
+title: Supporting complex formats - what we will and won't do, and what you can do to help
+categories: file-formats community
+---
+
+You may have noticed that a few months ago, we received an email asking us
+about when we expect to support 3D HISTECH .mrxs files. This sort of request
+isn’t particularly unusual and the reply gives an insight into one of the key 
+challenges we face.
+
+##Just because we don’t have a reader, doesn’t mean we haven’t done any work
+
+3D HISTECH .mrxs is an example of a complex format, the design of which does
+not make our work any easier. In fact, we can say with some confidence that
+the 3D HISTECH .mrxs file format is the most complex whole slide imaging file
+format we have ever encountered. We can say this because although we haven’t
+delivered a full reader for .mrxs—and there hasn’t been substantial public
```

Comment:
Is the hyphen here rendered as a hyphen or an emdash? I assume the latter is intended?